### PR TITLE
Updating Ticket prices

### PR DESCRIPTION
**Pull request by inovizz**

### CODE REVIEW

File: data/tickets.yml [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
 
-  - 
+  -
     name: Regular Ticket
-    details: Sale ends by 15 September, 2020 11:59 PM (Asia/Kolkata) OR completion of 2000 tickets
-    price: 199
+    details: Sale ends by 15 September, 2021 11:30 PM (Asia/Kolkata) OR completion of 800 tickets
+    price: 600
     category: conference
     link: ""
```
Comment (SaptakS) [inline data/tickets.yml:15]: Please add links to the townscripts ticket here and other tickets before release.

**Review by SaptakS — CHANGES_REQUESTED** [review]
In case you don't want to release some tickets, e.g. late bird or something, you can just remove that block and it won't show up. You can do the same for adding more ticket options

File: data/tickets.yml (hunk 1 above)
Comment (inovizz) [reply data/tickets.yml:15]: Sure, let me fix this.

**Comment by inovizz** [comment]
> In case you don't want to release some tickets, e.g. late bird or something, you can just remove that block and it won't show up. You can do the same for adding more ticket options

I've commented out the other categories which we are not launching for now. Will uncomment them when we have to release them on the website.

**Review by SaptakS — APPROVED** [review]
Looks good to me. Once it is ready to be launched, it can be merged and deployed.

**Comment by palnabarun** [comment]
Tried to close and reopen to see if Netlify rebuilds the site.

**Pull request merged**